Bluepill-runner/Bluepill-runner/main.m

```diff
@@ -73,7 +73,8 @@ int main(int argc, char * argv[]) {
         
         //Check if Bluepill compile time Xcode version is matched with Bluepill runtime Xcode version
         //Senario to prevent: Bluepill is compiled with Xcode 8, but runs with host installed with Xcode 9
-        if ([[BPUtils getXcodeRuntimeVersion] isEqualToString:@XCODE_VERSION]) {
+        //Only compare major version Exg. 9.1 == 9.1
+        if ([[[BPUtils getXcodeRuntimeVersion] substringToIndex:3] isEqualToString:[@XCODE_VERSION substringToIndex:3]]) {
             printf("Bluepill runtime version and compile time version are matched: %s\n", XCODE_VERSION);
         } else {
             fprintf(stderr, "ERROR: Bluepill runtime version %s and compile time version %s are mismatched\n", [[BPUtils getXcodeRuntimeVersion] UTF8String], XCODE_VERSION);
```

Comment:
we should also update the error msg to only use substring to index 3.. otherwise people would think you need the exact patch version